src/shared/plan/ScheduledPlanDetails/ScheduledPlanDetails.tsx

```diff
@@ -1,12 +1,23 @@
 import { format, fromUnixTime } from 'date-fns'
-import PropType from 'prop-types'
 
-export function getScheduleStart(scheduledPhase) {
+export function getScheduleStart(scheduledPhase: {
```

Comment:
thoughts on turning this into a shared interface for these two functions?

Reply:
Yeah, I like that - fixed!